caps/templates/caps/search_results.html

```diff
@@ -55,6 +66,16 @@ <h1 class="mb-3">Search inside plans</h1>
     <div class="col-md-8">
       {% if query %}
 
+      <div class="card ceuk-card ceuk-card--red ceuk-card--muted mb-3 mb-lg-4">
+        <div class="card-body mb-n3">
+          <h2>Search results for "{{ query }}"</h2>
+          {% if related_words %}
+            <p>Terms like the following were also used to improve the search: {% for term in related_words %}<span class="badge badge-secondary">{{ term }}</span> {% endfor %}</p>
```

Comment:
In your comment, you say:

> That means any terms that were actually present are highlighted at the top of the search results.

If that’s the case, why does this template say "Terms _like_ the following were also used…"? Isn’t it just "The following terms were also used…"?

Reply:
Also, no need for these to be marked up as `<span class="badge">`. Looks super heavy when there are dozens of them in a massive block. And people might expect to click them, because they look like little buttons.

Reply:
Yes, that was from an earlier bit where I was showing terms that weren't necessarily matching. Should be changed.

Reply:
now looks like this 
![image](https://user-images.githubusercontent.com/8157058/232528402-faeb3d9e-8850-4967-8baa-fc7075b92a31.png)